.gitignore

```diff
@@ -23,5 +24,4 @@ build
 *.idb
 *.exp
 
-example/addons/tiltfive/*.dll
-!extension/TiltFiveNDK/lib/*/*
\ No newline at end of file
+!extension/TiltFiveNDK/lib/*/*/*
```

Comment:
A newline at the end here would be a good idea, it's not strictly needed but git always complains about it